rascal-lsp/src/main/rascal/lang/rascal/tests/rename/Grammars.rsc

```diff
@@ -27,6 +27,7 @@ POSSIBILITY OF SUCH DAMAGE.
 module lang::rascal::tests::rename::Grammars
 
 import lang::rascal::tests::rename::TestUtils;
+import lang::rascal::lsp::refactor::Exception;
```

Comment:
Extra import but no use of the import? The same goes for imports in `ProjectOnDisk.rsc`.

Reply:
These imports fix errors regarding undefined `@expected` exceptions in these modules.